### DISCUSSION TITLE
Backport chains?

### DISCUSSION THREAD

**Discussion by humblemodulo** [wevm/viem#2185] [Idea / Feature Request]
Any chance we could get chains backported?  Currently still using Viem 1.22.x on an older repo, but need access to Amoy on an as-needed basis.  Could certainly hard-code it, but would be nice to simply have it available.

**Comment by tmm** [comment]
Feel free to open a PR with any chains you need

https://github.com/wevm/viem/blob/main/.github/CONTRIBUTING.md#chains
